### PR TITLE
Upgrade deps and rerun ffigen

### PR DESCRIPTION
**Pull request by nielsenko**

### CODE REVIEW

File: test/credentials_test.dart [hunk 1]
```diff
@@ -104,8 +104,6 @@ Future<void> main([List<String>? args]) async {
   baasTest('Email/Password - confirm user token expired', (configuration) async {
     final app = App(configuration);
     final authProvider = EmailPasswordAuthProvider(app);
-    String username = generateRandomEmail();
```
Comment (desistefanova) [inline test/credentials_test.dart:107]: Isn't this already deleted in main?

Comment (blagoev) [reply test/credentials_test.dart:107]: if it is it won't matter when it is merged

**Review by blagoev — APPROVED** [review]

**Review by desistefanova — APPROVED** [review]

File: test/credentials_test.dart (hunk 1 above)
Comment (nielsenko) [reply test/credentials_test.dart:107]: Yes, I'll rebase

**Comment by nielsenko** [comment]
I'll merge despite the Linux failures. Happens on current main as well, and will be investigated separately

**Pull request merged**